shopping_mall/backend/ai/agent/order_graph/graph.py

```
@@ -0,0 +1,113 @@
+    create_ticket,
+    handle_flow_cancel,
+    route_after_list_orders,
+    route_after_get_reason,
+    route_after_show_summary,
+    route_abort_check,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def build_order_graph(checkpointer) -> "CompiledStateGraph":
```

Ruff (0.15.10)

[error] Undefined name `CompiledStateGraph`

(F821)